src/main/java/Rizzler.java

```diff
@@ -23,6 +23,34 @@ public static void main(String[] args) {
                 case "list":
                     rizzlerSpeech.list(taskLog.getLog());
                     continue;
+                case "delete":
+                    rizzlerSpeech.say();
+                    if (userInputArr.length == 1) {
+                        rizzlerSpeech.say("what's the task number you want to delete?");
+                        rizzlerSpeech.say();
```

Comment:
I feel like you can abstract these repeated .say() calls to a Ui object which can handle this logic.